1-python-wrapper/wrapper/main.py

```diff
@@ -0,0 +1,33 @@
+from flask import Flask, request
+from model.model import load_most_recent_model
+import pandas as pd
+import logging
+
+
+logger = logging.getLogger(__name__)
+app = Flask(__name__)
+
+@app.route('/predict', methods=["GET", "POST"])
+def predict():
+    if request.method == "POST":
+        print(schema)
+        input = request.json
+        data = pd.DataFrame(input, index=[0])
+        return {"classification": str(model.predict(data)[0])}
+    else:
+        return "Please provide post request according to schema"
```

Comment:
I think we should nest this in an object as well. `{"error": "..."}`

Reply:
If I understand correctly, we will either way send a json back, either with the classification, or with the error, right?